.travis.yml

```diff
@@ -20,6 +22,20 @@ matrix:
       osx_image: xcode7
       env: PYTHON=3.5 PYTHONWARNINGS=ignore
 
+    - os: windows
+      language: shell  # 'language: python' is an error on Travis CI Windows
```

Comment:
What about just omitting `language`? Then the `language: generic` from the top of the file will be used.

Reply:
Please read thru https://docs.travis-ci.com/user/reference/windows  __generic__ is not on the __supported-languages__ list and using it causes the build to fail.

Reply:
https://chocolatey.org Is the tool that Travis builds their images with and it is bundled in the base image like homebrew is for their macOS images and apt is for their Linux images.  We can use https://chocolatey.org/search?q=Miniconda to install Miniconda.  I would prefer if we could first concentrate on getting the Ray test suite passing  on Windows.  After those tests pass, we can moving the build logic out of .travis.yml and into multiple files.